data/functions/utils.sql

```diff
@@ -5,6 +5,15 @@ begin
 end
 $$ language plpgsql immutable;
 
+create or replace function get_transaction_year(transaction_date timestamp, report_year numeric)
```

Comment:
Interesting, can you overload functions in Postgres like with C++/C# where you provide multiple method declarations that accept different arguments?